packages/mask/src/social-network-adaptor/twitter.com/injection/ToolboxHint_UI.tsx

```diff
@@ -48,6 +44,19 @@ export function ToolboxHintAtTwitter(props: { category: 'wallet' | 'application'
             </Text>
         )
     }, [buttonStyle.iconSize, textMarginLeft])
+
+    const handlerWatcher = () => {
+        setMini(!searchHomeLinkName().evaluate())
+    }
+
+    const onResize = debounce(handlerWatcher, 500)
```

Comment:
You should check https://github.com/streamich/react-use/blob/master/docs/useDebounce.md